PowerEditor/src/MISC/md5/md5Dlgs.cpp

```diff
@@ -32,7 +32,7 @@ intptr_t CALLBACK HashFromFilesDlg::run_dlgProc(UINT message, WPARAM wParam, LPA
 		{
 			NppDarkMode::autoSubclassAndThemeChildControls(_hSelf);
 
-			int fontDpiDynamicalHeight = NppParameters::getInstance()._dpiManager.scaleY(13);
+			int fontDpiDynamicalHeight = NppParameters::getInstance()._dpiManager.scaleY(15);
```

Comment:
I feel complete font can be removed. Let it take from system default.

Reply:
@SinghRajenM 
Do you mean that don't call CreateFontA ?

Reply:
@donho Yes.

Reply:
@SinghRajenM 
I do remember now why CreateFontA is called:
The default font in listbox is not proportional, hence "Courier New" was used by calling CreateFontA.
If you have other idea than CreateFontA, feel free to let me know.

Reply:
@donho 
Ok. If that is the case, then I think we should should be good with little bit increased font height. May be 15 (as in this PR) or 16.

Reply:
@SinghRajenM 

Could you HFONT `hFont = createFont(TEXT("Courier New"), 9, false, _hSelf);` instead of 
```
int fontDpiDynamicalHeight = NppParameters::getInstance()._dpiManager.scaleY(13);
HFONT hFont = ::CreateFontA(fontDpiDynamicalHeight, 0, 0, 0, 0, FALSE, FALSE, FALSE, ANSI_CHARSET,
				OUT_DEFAULT_PRECIS, CLIP_DEFAULT_PRECIS, DEFAULT_QUALITY,
				DEFAULT_PITCH | FF_DONTCARE, "Courier New");
```
to see if the font render better on your system?
If yes, please use `createFont` instead in this PR.

Reply:
Yes, it looks better with `HFONT hFont = createFont(TEXT("Courier New"), 9, false, _hSelf);`


![image](https://user-images.githubusercontent.com/14791461/196077876-c3d6fa48-6cf8-4eb4-938a-cfb9e574532b.png)